### PR TITLE
SW-2537 Support preselection in zone/plot selector

### PR DESCRIPTION
**Pull request by constanzauanini**

### CODE REVIEW

File: src/components/Plants/index.tsx [hunk 1]
```diff
@@ -141,13 +142,11 @@ export default function PlantsDashboard(props: PlantsDashboardProps): JSX.Elemen
       <Grid item xs={12}>
         <PageSnackbar />
       </Grid>
-      <Box ref={contentRef} display='flex' flexGrow={1}>
+      <Box ref={contentRef}>
```
Comment (karthikbtf) [inline src/components/Plants/index.tsx:145]: This change undoes my last fix.. add it back?

**Review by karthikbtf — APPROVED** [review]
Looks good, thanks for this enhancement.
One comment about a change

File: src/components/Plants/index.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/components/Plants/index.tsx:145]: Oh, good catch, I must have been during the merge

**Review by karthikbtf — APPROVED** [review]
Thanks 👍 looks good on vercel!
I do notice the 'x' icon showing up even if there is no selection - something we can clean up later maybe

**Pull request merged**